ex08/ex08.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+nums=( zeroes, ones, twoes, threes, fours, fives, sixs, sevens, eights, nines)                
+
+
+if [[ $# -gt 1 || $# -lt 1 ]]
+then
+  echo "Error.."
+  exit 1
+fi
+
+
+
+for i in {0..9}
+do
+  arr[$i]=$(grep -o $i $1 | wc -l)
+  result+=' '${arr[$i]}' '${nums[$i]}
+done
+echo $result 
```

Comment:
All tests except first one are failed, i commit fixed test

Reply:
fixed this one too